### PR TITLE
update kube dependencies to v1.25

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

> /kind cleanup

```release-note
This release update kubernetes module dependencies to v1.25
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,8 +4,7 @@ go 1.16
 
 require (
 	github.com/container-storage-interface/spec v1.6.0
-	github.com/go-logr/logr v1.2.3 // indirect
-	github.com/golang/mock v1.5.0
+	github.com/golang/mock v1.6.0
 	github.com/kubernetes-csi/csi-lib-utils v0.11.0
 	github.com/kubernetes-csi/csi-test/v4 v4.0.0-20200806214950-555d70a11a8b
```
Comment (xing-yang) [inline go.mod:9]: Can you update this to v5.0.0?
https://github.com/kubernetes-csi/csi-test/releases/tag/v5.0.0

Comment (humblec) [reply go.mod]: @xing-yang done.. ptal.. thanks

File: go.mod [hunk 2]
```diff
@@ -4,16 +4,16 @@ go 1.16
...
+	github.com/kubernetes-csi/csi-test/v5 v5.0.0
 	github.com/prometheus/client_golang v1.12.2 // indirect
 	github.com/prometheus/common v0.37.0 // indirect
 	github.com/prometheus/procfs v0.8.0 // indirect
+	golang.org/x/net v0.0.0-20220921203646-d300de134e69 // indirect
 	google.golang.org/genproto v0.0.0-20220804142021-4e6b2dfa6612 // indirect
 	google.golang.org/grpc v1.48.0
 	google.golang.org/protobuf v1.28.1 // indirect
-	k8s.io/component-base v0.24.3 // indirect
+	k8s.io/component-base v0.25.0 // indirect
```
Comment (xing-yang) [inline go.mod:17]: 0.25.2

Comment (humblec) [reply go.mod]: @xing-yang done. ptal..

**Comment by humblec** [comment]
@xing-yang done.. ptal.

**Pull request merged**